app/com/linkedin/drelephant/analysis/AnalyticJob.java

```diff
@@ -331,6 +343,36 @@ public AppResult getAnalysis() throws Exception {
 
     // Retrieve information from job configuration like scheduler information and store them into result.
     InfoExtractor.loadInfo(result, data);
+    if (result.queueName.toLowerCase().equals("ump_normal") || result.queueName.toLowerCase().equals("ump_hp")) {
```

Comment:
Giving out internal queue configuration. At a minimum make this configurable. Better model the conditional execution as an interface? Applies elsewhere too